tests/ga/test_extension.py

```diff
@@ -16,13 +16,20 @@
 #
 
 import os.path
+import unittest
 
 from datetime import timedelta
 
 from azurelinuxagent.ga.monitor import get_monitor_handler
 from nose.plugins.attrib import attr
-from tests.protocol.mockwiredata import *
-
+from tests.protocol.mockwiredata import DATA_FILE, DATA_FILE_EXT_AUTOUPGRADE, \
+    DATA_FILE_EXT_AUTOUPGRADE_INTERNALVERSION, DATA_FILE_EXT_DELETION, DATA_FILE_EXT_INTERNALVERSION, \
+    DATA_FILE_EXT_ROLLINGUPGRADE, DATA_FILE_EXT_SEQUENCING, DATA_FILE_EXT_SINGLE, DATA_FILE_NO_EXT, \
+    DATA_FILE_EXT_NO_PUBLIC, DATA_FILE_EXT_NO_SETTINGS, DATA_FILE_MULTIPLE_EXT, WireProtocolData
```

Comment:
since this list is likely to keep growing, maybe instead do
```
import tests.protocol.mockwiredata 
```
and then qualify each reference? e.g
```
mockwiredata.DATA_FILE
```

Reply:
Yeah, it was getting slightly out of hand. I'll update this on this PR.